galaxy_old.yml

```diff
@@ -0,0 +1,56 @@
+---
+- hosts: galaxy_old
+  become: true
+  become_user: root
+  pre_tasks:
+    - name: Fix Hostname
+      hostname:
+        name: "{{ inventory_hostname }}"
+    - name: Install Dependencies
+      package:
+        name: ['acl', 'bzip2', 'git', 'make', 'python3-psycopg2', 'tar', 'virtualenv', 'cgroup-tools', 'libldap2-dev', 'libsasl2-dev']
+    - name: Create the second storage directory
+      file:
+        owner: galaxy
+        group: galaxy
+        path: //mount
+        state: directory
+        mode: '0755'
+    - debug:
+        msg: "Galaxy version is {{ galaxy_commit_id }}"
+
+
+  handlers:
+    - name: Restart Galaxy
+      systemd:
+        name: galaxy
+        state: restarted
+  roles:
+    - avans
+    - ansible-htcondor
+    - geerlingguy.nfs
+    - galaxyproject.postgresql
+    - role: natefoo.postgresql_objects
+      become: true
+      become_user: postgres
+    - geerlingguy.pip
+    - gantsign.golang
+    - galaxyproject.galaxy
+    - role: uchida.miniconda
+      become: true
+      become_user: "{{ galaxy_user.name }}"
+    - usegalaxy_eu.rabbitmq
+    - usegalaxy_eu.galaxy_systemd
+    - usegalaxy_eu.gxadmin
+    - hxr.monitor-cluster
+    - galaxyproject.nginx
+    - galaxyproject.cvmfs
+    - influxdata.chrony
+    - dj-wasabi.telegraf
+    - usegalaxy_eu.tiaas2
+    - geerlingguy.docker
+    - 
+    - role: usegalaxy_eu.gie_proxy
+      environment:
+        SUDO_USER: root
+diff --git a/hosts b/hosts

```

Comment:
```suggestion
```

Reply:
That seems alot more logical. Sorry I misunderstood, for some reason I was thinking that we keep the old galaxy alive.

Reply:
in that case:

- RabbitMQ should be on the new galaxy server
- just pulsar on the old
- job conf won't need an `if` (but good that you learned how to do that! very nice.)

Reply:
weird question. but how do we resolve exaclt that the new servers wont communicate with the old servers?

Reply:
that's why we're using pulsar, the old *can* contact the new, but not the other way around. So we put rabbitMQ on the new server, and the old server connects to it and waits for jobs to show up

Reply:
ahh okay, I thought it would fail because it was unable to send the job

Reply:
This is where I wish we had the infra diagram! 

yeah, so it'll be roughly

```
galaxy.atgm.avans.nl:
  - galaxy (sends jobs to rabbitmq)
  - rabbitmq

old:
 - pulsar, connects to RabbitMQ and just talks to it
```

Reply:
comments are resolved